Packs/Tenable_io/Integrations/Tenable_io/README.md

```diff
@@ -1,1462 +1,487 @@
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| TenableIO.Scan.Id | unknown | The unique id of the scan. | 
+| TenableIO.Scan.Status | unknown | The status of the scan \(completed, aborted, imported, pending, running, resuming, canceling, cancelled, pausing, paused, stopping, stopped\). | 
+
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
+
+
+
+### tenable-io-add-tags
+***
+Add tags to Tenable tenant
+
+
+#### Base Command
+
+`tenable-io-add-tags`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| payload | Payload for REST call | Required | 
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
+
+
+
+### tenable-io-resume-scans
+***
+Resume all scans inputted as an array
+
+
+#### Base Command
+
+`tenable-io-resume-scans`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| scanIds | Comma separated scan ID's | Required | 
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
+
+
+
+### tenable-io-pause-scans
+***
+Pauses all scans imputted as ana rray
+
+
+#### Base Command
+
+`tenable-io-pause-scans`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| scanIds | Comma separated Scan ID's  | Required | 
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
+
+
+
+### tenable-io-launch-scans
+***
+Launches multiple scans
+
+
+#### Base Command
+
+`tenable-io-launch-scans`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| scan_ids | Comma separated list of scan ID's | Required | 
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
+
+
+
+### tenable-io-create-scan
+***
+Creates a new scan from a JSON body
+
+
+#### Base Command
+
+`tenable-io-create-scan`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| scanSettings | JSON data to build the scan from | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Tenable.IO.NewScan.Id | string | ID of the scan that was just created | 
+| Tenable.IO.NewScan.ScanIds | string | List of IDs created | 
+
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
+
+
+
+### tenable-io-check-temaplates
+***
+Returns Info about scan templates
+
+
+#### Base Command
+
+`tenable-io-check-temaplates`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| templates | unknown | The temaplates | 
+
+
+#### Command Example
```

Comment:
@barchen1 Have you tried looking into the command examples?

Reply:
ADDRESSED
Update 02.23.2021 1034 HRS EST:
We will need assistance with generating the README.md file.  It looks like the demisto-sdk command that generates the README.md file requires the ability to run the commands associated with the integration.  We have to use Luminate to connect to our Demisto instance and would require some technical assistance to get that configured.